<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>992</width>
    <height>685</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>200</y>
     <width>971</width>
     <height>465</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTextEdit" name="NO1"/>
    </item>
    <item row="1" column="1">
     <widget class="QTextEdit" name="SEAT5"/>
    </item>
    <item row="4" column="2">
     <widget class="QTextEdit" name="SEAT13"/>
    </item>
    <item row="3" column="3">
     <widget class="QTextEdit" name="SEAT17"/>
    </item>
    <item row="0" column="3">
     <widget class="QTextEdit" name="SEAT14"/>
    </item>
    <item row="3" column="2">
     <widget class="QTextEdit" name="SEAT12"/>
    </item>
    <item row="1" column="2">
     <widget class="QTextEdit" name="SEAT10"/>
    </item>
    <item row="0" column="1">
     <widget class="QTextEdit" name="SEAT4"/>
    </item>
    <item row="1" column="0">
     <widget class="QTextEdit" name="SEAT1"/>
    </item>
    <item row="3" column="1">
     <widget class="QTextEdit" name="SEAT7"/>
    </item>
    <item row="4" column="1">
     <widget class="QTextEdit" name="SEAT8"/>
    </item>
    <item row="3" column="5">
     <widget class="QTextEdit" name="SEAT21"/>
    </item>
    <item row="4" column="5">
     <widget class="QTextEdit" name="NO4"/>
    </item>
    <item row="0" column="2">
     <widget class="QTextEdit" name="SEAT9"/>
    </item>
    <item row="2" column="2">
     <widget class="QTextEdit" name="SEAT11"/>
    </item>
    <item row="2" column="3">
     <widget class="QTextEdit" name="SEAT16"/>
    </item>
    <item row="1" column="3">
     <widget class="QTextEdit" name="SEAT15"/>
    </item>
    <item row="4" column="3">
     <widget class="QTextEdit" name="SEAT18"/>
    </item>
    <item row="0" column="5">
     <widget class="QTextEdit" name="NO3"/>
    </item>
    <item row="1" column="5">
     <widget class="QTextEdit" name="SEAT19"/>
    </item>
    <item row="2" column="5">
     <widget class="QTextEdit" name="SEAT20"/>
    </item>
    <item row="3" column="0">
     <widget class="QTextEdit" name="SEAT3"/>
    </item>
    <item row="4" column="0">
     <widget class="QTextEdit" name="NO2"/>
    </item>
    <item row="2" column="0">
     <widget class="QTextEdit" name="SEAT2"/>
    </item>
    <item row="2" column="1">
     <widget class="QTextEdit" name="SEAT6"/>
    </item>
    <item row="0" column="4">
     <widget class="QTextEdit" name="SEAT26"/>
    </item>
    <item row="1" column="4">
     <widget class="QTextEdit" name="SEAT22"/>
    </item>
    <item row="2" column="4">
     <widget class="QTextEdit" name="SEAT23"/>
    </item>
    <item row="3" column="4">
     <widget class="QTextEdit" name="SEAT24"/>
    </item>
    <item row="4" column="4">
     <widget class="QTextEdit" name="SEAT25"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>312</x>
     <y>60</y>
     <width>291</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>추첨 시작</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
